src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -41,15 +41,25 @@ public MarkdownV2Renderer(ParserMode mode, int maxSyntaxWidth)
 
         public string MamlModelToString(MamlCommand mamlCommand, bool skipYamlHeader)
         {
-            return MamlModelToString(mamlCommand, null, skipYamlHeader);
+            return MamlModelToString(mamlCommand, null, skipYamlHeader, false);
```

Comment:
please add parameter names for readability, when passing constants as parameters: i.e. `false` -> `detailedInput : false`.

The existing codebase doesn't always follow that rule, good occasion to fix some old places as well, if you don't mind.